packages/aura/src/palette.css

```diff
@@ -0,0 +1,45 @@
+:where(html) {
+  --aura-red: light-dark(
```

Comment:
General question: why did you specify `--aura-red` instead of `--vaadin-red`? 

My assumption would have been like this: I can re-use those variables inside my theme that extends aura - meaning I would always have to remember that they are named `--aura` instead of `--vaadin` like everything else. This also comes with the problem that I can't switch e.g. between aura and another theme you create in the future, which might use `--lorem`?  (I know there is `--lumo`.. but this feels like a deprecated thing.. or?)

Reply:
The only/main reason because these properties are defined by the Aura theme.

Reply:
That would leave the existing Lumo properties in a weird place, though.

Reply:
Lumo is "deprecated" anyway, or? I would vote for `--vaadin`, but that's up for you and the others to decide :) from a user's perspective it feels "better" to only remember `--vaadin regardless of which theme we use

Reply:
Lumo isn't entirely deprecated. The shadow DOM implementation is. We are planning on refactoring/rewriting it in 25 minors from shadow DOM styles to light DOM. That could be a place where we rename the properties as well.

I agree, it would be nice to have just a single namespace for these. Somehow it just feels academically correct to have a theme-specific prefix.